src/app/donation-start/donation-start-form/donation-tipping-slider/donation-tipping-slider.component.spec.ts

```diff
@@ -32,13 +32,42 @@ describe('DonationTippingSliderComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should set the percentage defaults according to tipAmounts', () => {
-    const slider =  new DonationTippingSliderComponent({
-      listen: () => { },
-    } as unknown as Renderer2);
-    slider.donationAmount = 100;
-    slider.calcAndSetPercentage();
-    expect(slider.derivedPercentage).toBe(12.5);
-  });
+  usingDataIt('should have default percentage according to donation size', [
+  // [donation , expected default percentage]
+    [0, 15],
+    [99, 15],
+    [100, 12.5],
+    [299, 12.5],
+    [300, 10],
+    [999, 10],
+    [1000, 7.5],
+    ], function (name: string, args: Array<any>){
```

Comment:
would be nice to get rid of the `any`s, I feel like there should be a way to get rid of it with generics. But it's not as important for tests as it is in production.